graphcanvas/graph_view.py

```diff
@@ -115,9 +119,9 @@ def _layout_changed(self, new):
         self._canvas.style = new
 
     def _on_hover(self, label):
-        print "hovering over:", label
+        print(u"hovering over: {}".format(text(label)))
```

Comment:
On Python 2, this only works if `label` is pure ASCII: if `label` is non-ASCII of type `str`, there'll be a `UnicodeDecodeError` from the formatting operation, and if `label` is non-ASCII of type `unicode`, there'll be a `UnicodeEncodeError` from the `text(label)` conversion.